packages/blockchain-wallet-v4-frontend/src/data/goals/sagas.ts

```diff
@@ -202,8 +202,8 @@ export default ({ api, coreSagas, networks }) => {
       return yield call(defineSwapGoal)
     }
 
-    // /#/open/interest
-    if (startsWith(DeepLinkGoal.INTEREST, pathname)) {
+    // /#/open/rewards
+    if (startsWith(DeepLinkGoal.REWARDS, pathname)) {
```

Comment:
where/how is this deeplink/goal used? changing this may break external references... 🤷

Reply:
yeah good point, i'll follow up with Pedro about this. He added them initially.